### DISCUSSION TITLE
Still not getting 2500mbps on my DS223 😪

### DISCUSSION THREAD

**Discussion by ORah82** [bb-qq/r8152#437] [Q&A] [answered]
Installed 2.18.1-1/r8152-rtd1619b-2.18.1-1_7.2.spk with a  Club3D 2.5 Gigabit USB adapter (CAC-1420). Still not getting higher speeds. 

![Screenshot 2024-10-03 230014](https://github.com/user-attachments/assets/a84247cd-70ba-49ee-ab74-65758d44b837)
<img width="622" alt="Screenshot 2024-10-03 225835" src="https://github.com/user-attachments/assets/eb77d22d-bfe1-4072-a580-49ed28ac1572">

Any tips would be appreciated.

**Comment by bb-qq** [comment]
Did you check this page?
https://github.com/bb-qq/r8152/wiki/Troubleshooting#performance-issues

**Reply by ORah82** [reply, marked as answer]
I think it's my switch. I need a switch with 2.5G port or more.

Thanks for all you do!
